astropy/modeling/bounding_box.py

```diff
@@ -20,7 +19,9 @@
 __all__ = ["ModelBoundingBox", "CompoundBoundingBox"]
 
 
-_BaseInterval = namedtuple("_BaseInterval", "lower upper")
+class _BaseInterval(NamedTuple):
```

Comment:
Does ``_BaseInterval`` need to exist?

Reply:
No, but a change like that should be done separately.